pom.xml

```diff
@@ -22,7 +22,7 @@
 	</organization>
 	<licenses>
 		<license>
-			<name>Simplified BSD License</name>
+			<name>GPL</name>
```

Comment:
We have to stick to the license names supported by `maven-license-plugin`: http://www.mojohaus.org/license-maven-plugin/examples/example-license-list.html
Let's change this to `gpl_v3`.